xmtp_mls/src/groups/mod.rs

```diff
@@ -706,6 +714,36 @@ impl<ScopedClient: ScopedGroupClient> MlsGroup<ScopedClient> {
         Ok(message_id)
     }
 
+    /// Helper function to extract queryable content fields from a message
+    fn extract_queryable_content_fields(message: &[u8]) -> QueryableContentFields {
+        let default = QueryableContentFields {
+            content_type: ContentType::Unknown,
+            version_major: 0,
+            version_minor: 0,
+            authority_id: "unknown".to_string(),
+        };
+
+        // Return early with default if decoding fails or type is missing
+        let content_type_id = match EncodedContent::decode(message)
```

Comment:
Using `Default::default` this can probably be cleaner


you can use [Result::inspect_err](https://doc.rust-lang.org/std/result/enum.Result.html#method.inspect_err) to throw the log message.

then use [Result::unwrap_or_default](https://doc.rust-lang.org/std/result/enum.Result.html#method.unwrap_or_default)

then `EncodedContent::decode(message).map(|c| c.type_id).unwrap_or_default()` with the `Default` trait.